c2corg_ui/static/js/maincontroller.js

```diff
@@ -76,3 +91,17 @@ app.MainController.prototype.isPath = function(path) {
 app.MainController.prototype.animateHeaderIcon = function(e) {
   app.utils.animateHeaderIcon(e);
 }
+
+
+/**
+ * @export
+ */
+app.MainController.prototype.redirect = function(url) {
```

Comment:
I'm not sure how you want to be more explicit about a 'redirect' function than calling it 'redirect' ? :kissing: